src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int ZERO_INDEX = 0;
+    static final int FIRST_INDEX = 1;
+    static final String CSV_SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String readString = readFromFile(fromFileName);
+        String reportString = createReport(readString);
```

Comment:
Pay MORE attention to the names.
```suggestion
        String report = createReport(readString);
```